docs/EmbeddedSwift/ABI.md

```diff
@@ -0,0 +1,29 @@
+# Embedded Swift -- ABI
+
+**⚠️ Embedded Swift is experimental. This document might be out of date with latest development.**
+
+**‼️ Use the latest downloadable 'Trunk Development' snapshot from swift.org to use Embedded Swift. Public releases of Swift do not yet support Embedded Swift.**
+
+For an introduction and motivation into Embedded Swift, please see "[A Vision for Embedded Swift](https://github.com/apple/swift-evolution/blob/main/visions/embedded-swift.md)", a Swift Evolution document highlighting the main goals and approaches.
+
+## Calling convention of Embedded Swift
+
+As of today, Embedded Swift has identical calling convention to full Swift. However, this does not need to continue in the future, and there should not be expectations that the ABI of Embedded Swift is compatible with full Swift.
```

Comment:
It might be good to discuss the interoperation calling conventions (e.g. mentioning the ability to interface with C/C++)